MetadataMigration/src/main/java/org/opensearch/migrations/MetadataMigration.java

```diff
@@ -77,7 +77,7 @@ public static void main(String[] args) throws Exception {
                 result = meta.evaluate(evaluateArgs).execute(context);
                 break;
         }
-        log.info(result.toString());
+        log.info(result.asCliOutput());
```

Comment:
This can break if cliOutput contains braces, let's do `log.atInfo().setMessage("{}").addArgument(...).log()`